### ISSUE TITLE
Excessive Gradually Increasing DNS Requests to Torrent Trackers

### ISSUE THREAD

**Issue by otuva** [Stremio/stremio-bugs#450]
**Describe the bug**
After using stremio for a while. With pihole I can observe DNS requests to torrent trackers increasing in frequency. 
Like 1k req/m
In the screenshot, the slope drops suddenly when I close stremio completely. But if it stays in the background, or I'm using it actively, requests continue
As you can see from the screenshot, client activity correlates and other clients are negligible
I'm using the latest stable flatpak version

**To Reproduce**
1. Spend time on stremio
2. Observe on pihole

**Expected behavior**
It shouldn't send that many 

**Screenshots**
![image](https://user-images.githubusercontent.com/67955086/223208535-26a8e1e9-d6cb-4c69-8ddb-1935783573d9.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: FIrefox
 - Stremio version: 4.4.159 (Flatpak)

**Comment by otuva** [comment]
Yes, I was able to confirm this happens while using both AUR (stremio-beta) and deb versions.

**Comment by TRtomasz** [comment]
@jaruba  @core1024 what do you think can be causing this?

**Comment by desuwulf** [comment]
Experiencing the same here.

**Comment by desuwulf** [comment]
this appears to be caused by a misbehaving addon. for anyone experiencing this: make sure that torrenting addons are not causing this. I suspect there is some unusual cache interaction for the app.

**Comment by TRtomasz** [comment]
Closing this issue as we have released new app version https://blog.stremio.com/stremio-5-released-for-linux/
